packages/core/src/index.js

```diff
@@ -1,6 +1,8 @@
 import './global/js/polyfills';
 import boot from './global/js/boot';
+import settingsObj from './global/js/settings';
```

Comment:
@adamraider Not sure a better way to do this. Jest was complaining about default exports

Reply:
`export {default as name1} from '...';`
try that